### ISSUE TITLE
Fix allow-empty-input CLI flag

### ISSUE THREAD

**Issue by fisker** [stylelint/stylelint#4028]
> Clearly describe the bug

Invalid option "`--allow-empty-input`", reports when i run 
`stylelint empty/*.css --allow-empty-input`

> Which rule, if any, is the bug related to?

> What CSS is needed to reproduce the bug?

> What stylelint configuration is needed to reproduce the bug?

> Which version of stylelint are you using?

`10.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

"CLI with `stylelint empty/*.css --allow-empty-input`"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

> What did you expect to happen?

"No error reported."

> What actually happened (e.g. what warnings or errors did you get)?

```
Invalid option "--allow-empty-input".
```

**Comment by jeddy3** [comment]
@fisker Thanks for the report and for using the template.

I can reproduce this.

**Comment by fisker** [comment]
thanks.

btw, the template is really long, to report just one command
